location/location.c

```diff
@@ -78,21 +91,17 @@ void getLocation(le_timer_Ref_t timerRef) {
 /**
  * Perform all required setup
  *
- * Note that we run this on a timer to avoid
- * blocking up the main (only) thread. If this
- * was run in a while(true) that sleeps,
- * the IPC caller would be blocked indefinitely
+ * Note that we get the position from a timer to avoid blocking up the main
+ * (only) thread. If this was run in a while(true) that sleeps, the IPC caller
+ * would be blocked indefinitely.
  */
-le_result_t gps_init() {
+COMPONENT_INIT {
```

Comment:
I personally liked having this in a sub-routing since we may have a few init sub-routines down the road.

Reply:
done